sdcm/cluster.py

```diff
@@ -2957,6 +2961,16 @@ def collect_logs(self, storing_dir):
                 shutil.copy(node.database_log, storing_dir_for_node)
         return storing_dir
 
+    def get_seed_selected_by_reflector(self, node=None):
+        """
+        Check if reflector updated the scylla.yaml with selected seed IP
+        """
+        if not node:
+            node = self.nodes[0]  # pylint: disable=no-member
+
+        seed_nodes_ips = node.extract_seeds_from_scylla_yaml()
+        return [n.ip_address for n in self.nodes if n.ip_address in seed_nodes_ips]  # pylint: disable=no-member
```

Comment:
why you don't just return `seed_nodes_ips`?

Reply:
added comment